.github/workflows/auto-merge-bot.yml

```diff
@@ -0,0 +1,17 @@
+name: Auto Merge Bot
+
+on:
+  # GitHub considers PRs as issues
+  issue_comment:
+    types: [created]
+
+jobs:
+  set-auto-merge:
+    runs-on: ubuntu-latest
+    # Important! This forces the job to run only on Pull Requests
+    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '/bot') }}
```

Comment:
I guess, it's the right time to question `/bot` part, and consider `/merge`

Reply:
I left this for a future task: https://github.com/paritytech/auto-merge-bot/issues/8

As I need to change a couple of things and thought it would be better there